src/auth/session/storage/custom.ts

```diff
@@ -70,11 +82,58 @@ export class CustomSessionStorage implements SessionStorage {
 
   public async deleteSession(id: string): Promise<boolean> {
     try {
-      return await this.deleteCallback(id);
+      return await this.deleteSessionCallback(id);
     } catch (error) {
       throw new ShopifyErrors.SessionStorageError(
         `CustomSessionStorage failed to delete a session. Error Details: ${error}`,
       );
     }
   }
+
+  public async deleteSessions(ids: string[]): Promise<boolean> {
+    if (this.deleteSessionsCallback) {
+      try {
+        return (await this.deleteSessionsCallback(ids)) as boolean;
```

Comment:
If the function doesn't return `undefined`, we probably won't need to cast this.

Reply:
👍🏻